src/app/components/Actionbutton/Actionbutton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './Actionbutton.module.css';
+
+export type ActionButtonProps = {
+  children: React.ReactNode;
+  type: 'button' | 'submit';
+  style: 'primary' | 'secondary';
+  onClick?: () => void;
+};
+
+const ActionButton = ({
+  children,
+  type,
+  style,
+  onClick,
+}: ActionButtonProps): JSX.Element => {
+  const styleClass = (): string => {
```

Comment:
A better name in my opinion would be `getStyles`

Reply:
I deleted the function and use a Map instead